fixtures/tracks/animal/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
I will be happy to create the PR to fix the fixtures, then reapply #12. coming soon.